manager/manager.go

```diff
@@ -233,7 +233,6 @@ func (self *manager) Start() error {
 		glog.Errorf("Registration of the raw container factory failed: %v", err)
 	}
 
-	// FIXME - delete?
```

Comment:
I didn't mean to include it in https://github.com/google/cadvisor/pull/1190 so I removed it here.